libunwind/include/__libunwind_config.h

```diff
@@ -180,6 +180,7 @@
 #endif
 #define _LIBUNWIND_HIGHEST_DWARF_REGISTER                                      \
   _LIBUNWIND_HIGHEST_DWARF_REGISTER_LOONGARCH
+#elif defined(__wasm__)
```

Comment:
Why don't we define `_LIBUNWIND_CURSOR_SIZE` and friends on wasm?

Reply:
What is the correct value here? I mean if you give users a number and they may make wrong assumptions here.